bigbluebutton-html5/imports/ui/components/modal/base/styles.scss

```diff
@@ -22,9 +22,7 @@
 
 .overlay {
   z-index: 1000;
-  // background: transparentize($color-white, .35);
-  background: #fff;
-  // background: transparentize($color-gray-dark, .40);
+  background: transparentize($color-gray-dark, .40);
```

Comment:
They're all modal. Why would the audio modals have a different look from every other modal in the client?